shared/cryptoutil/cryptoUtil.ts

```diff
@@ -87,7 +307,7 @@ export function encodeB64(str: string) {
  *  use the isCustomKeystore constant in the app lifeycle instead.
  */
 function isCustomSecureKeystore() {
-  return !isIOS() ? RNSecureKeystoreModule.deviceSupportsHardware() : false;
+  return !isIOS() ? RNSecureKeystoreModule.deviceSupportsHardware() : true;
```

Comment:
reverse the condition
if isAndroid()  ? RNSecureKeystoreModule.deviceSupportsHardware() : true;